public/components/security/main.tsx

```diff
@@ -56,49 +66,60 @@ export const WzSecurity = compose(
   const dispatch = useDispatch();
 
   // Get the initial tab when the component is initiated
-  const securityTabRegExp = new RegExp(`tab=(${tabs.map(tab => tab.id).join('|')})`);
+  const securityTabRegExp = new RegExp(`tab=(${tabs.map((tab) => tab.id).join('|')})`);
   const tab = window.location.href.match(securityTabRegExp);
 
-  const [selectedTabId, setSelectedTabId] = useState(tab && tab[1] || 'users');
-
+  const [selectedTabId, setSelectedTabId] = useState((tab && tab[1]) || 'users');
 
   const listenerLocationChanged = () => {
     const tab = window.location.href.match(securityTabRegExp);
-    setSelectedTabId(tab && tab[1] || 'users')
-  }
+    setSelectedTabId((tab && tab[1]) || 'users');
+  };
 
   const checkRunAsUser = async () => {
     const currentApi = AppState.getCurrentAPI();
     try {
-      const ApiCheck = await GenericRequest.request('POST',
-        '/api/check-api',
-        currentApi
-      );
+      const ApiCheck = await GenericRequest.request('POST', '/api/check-api', currentApi);
       return ApiCheck.data.allow_run_as;
-
     } catch (error) {
-      ErrorHandler.handle(error, 'Error checking the current API');
+      throw new Error(error);
```

Comment:
I think is better only `throw` the error, without the `new`. Except, when set a custom message error.